src/widget/wtrackmenu.cpp

```diff
@@ -914,6 +917,17 @@ class ImportMetadataFromFileTagsTrackPointerOperation : public mixxx::TrackPoint
 
 } // anonymous namespace
 
+void WTrackMenu::slotUpdateReplaygain() {
+    if (!m_pTrack) {
```

Comment:
Why not VERIFY_OR_DEBUG_ASSERT which would be more explicit and reveal the line of code? 

This log message carries no useful information. If this case is expected, then the message is not needed (until we switch to qInfo() for regular logs qDebug() must only be used for operational logs). If it is an error the the debug level is too low.

Reply:
done